bin/node-template/node/src/command.rs

```diff
@@ -102,10 +103,14 @@ pub fn run() -> sc_cli::Result<()> {
 			if cfg!(feature = "runtime-benchmarks") {
 				let runner = cli.create_runner(cmd)?;
 
-				runner.sync_run(|config| cmd.run::<Block, service::ExecutorDispatch>(config))
+				runner.sync_run(|config| match cmd {
+					BenchmarkCmd::Pallet(cmd) =>
+						cmd.run::<Block, service::ExecutorDispatch>(config),
+					_ => Err("Sub-command is not supported".into()),
+				})
 			} else {
-				Err("Benchmarking wasn't enabled when building the node. You can enable it with \
-				     `--features runtime-benchmarks`."
+				Err("Benchmarking wasn't enabled when building the node. \
```

Comment:
```suggestion
				Err("Runtime benchmarking wasn't enabled when building the node. \
```

Since some benchmarking will work without the feature flag right?

Reply:
Yes, I will also move that check into the `BenchmarkCmd::Pallet` match arm.